### PR TITLE
[LETS-653] [LETS-528] Initialize ps_Gl only when it is needed

### PR DESCRIPTION
**Pull request by wrlawodms**
http://jira.cubrid.org/browse/LETS-653
http://jira.cubrid.org/browse/LETS-528 

Use unique_ptr for `ps_Gl` and initialize it only when needed. See the Jira issue.

### CODE REVIEW

**Review by hornetmj — APPROVED** [review]
LGTM.

**Review by joohok — APPROVED** [review]

File: src/server/page_server.hpp [hunk 1]
```diff
@@ -249,7 +249,7 @@ class page_server
     pts_mvcc_tracker m_pts_mvcc_tracker;
 };
 
-extern page_server ps_Gl;
+extern std::unique_ptr<page_server> ps_Gl;
```
Comment (cristiarg) [inline src/server/page_server.hpp]: Could we move this in `server_type.h` - similar to where `ts_Gl` is?
The include clauses would be leaner.

**Comment by cristiarg** [comment]
@wrlawodms 
Please also reference http://jira.cubrid.org/browse/LETS-528 .
Both in the title of the PR and in the comment section:
```
[LETS-653][LETS-528] Initialize ps_Gl only when it is needed
```

File: src/server/page_server.hpp (hunk 1 above)
Comment (wrlawodms) [reply src/server/page_server.hpp]: Thank you, I missed it.
It's moved in server_type.hpp/cpp inside SERVER_MODE.

**Pull request merged**